network-api/networkapi/urls.py

```diff
@@ -65,6 +65,9 @@
 
     # Wagtail Footnotes package
     path("footnotes/", include(footnotes_urls)),
+
+    # redirect /pt to /pt-BR. See https://github.com/mozilla/foundation.mozilla.org/issues/5993
+    re_path(r'^pt/(?P<rest>.*)', RedirectView.as_view(url='/pt-BR/%(rest)s')),
```

Comment:
@Pomax I need a hand here. With this line, I can get /pt to redirect to /pt-BR but it doesn't carry over query string. 🤔

Reply:
Easy to get confused by: query string handling is behaviour of the RedirectView, so you can trigger it by passing `query_string=True` as part of the as_view() function call:

```suggestion
    re_path(r'^pt/(?P<rest>.*)', RedirectView.as_view(url='/pt-BR/%(rest)s', query_string=True)),
```

Reply:
omgggggg 🤯